content/webapp/views/components/WorkCard/WorkCards.tsx

```diff
@@ -0,0 +1,61 @@
+// Make default when newOnlineListingPage becomes default
+import { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { WorkBasic } from '@weco/content/services/wellcome/catalogue/types';
+import WorkCardAPI from '@weco/content/views/components/WorkCard/WorkCard.API';
+
+const Works = styled.div`
+  ${props => `
+    --gap: ${props.theme.gutter.small}px;
+  `}
+
+  ${props =>
+    props.theme.media('medium')(`
+    --gap: ${props.theme.gutter.medium}px;
+  `)}
+
+  ${props =>
+    props.theme.media('large')(`
+    --gap: ${props.theme.gutter.large}px;
+  `)}
+  
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  gap: var(--gap);
+`;
+
+const WorkContainer = styled.div`
+  flex: 0 0 calc(100% - var(--gap));
+
+  ${props =>
+    props.theme.media('medium')(`
+    flex: 0 0 calc(50% - var(--gap));
+  `)}
+
+  ${props =>
+    props.theme.media('large')(`
+    flex: 0 0 calc(25% - var(--gap));
+  `)}
+`;
+
+type Props = {
+  works: WorkBasic[];
+};
+
+const WorkCards: FunctionComponent<Props> = ({ works }) => {
+  if (works.length === 1) return <WorkCardAPI item={works[0]} />;
```

Comment:
Looks like we currently only render this component if the works array length is greater than 0, but wonder if it is worth handling that case within the component too (i.e. returning null if the length is 0).